integrations/spark-extensions-basetests/src/main/java/org/projectnessie/spark/extensions/AbstractNessieSparkSqlExtensionTest.java

```diff
@@ -563,7 +563,7 @@ private void mergeReferencesInto(String query)
   void mergeReferences() throws NessieConflictException, NessieNotFoundException {
     List<SparkCommitLogEntry> resultList = createBranchCommitAndReturnLog();
     sql("USE REFERENCE %s IN nessie", refName);
-    sql("MERGE BRANCH IN nessie");
+    sql("MERGE BRANCH %s INTO main IN nessie", refName);
```

Comment:
Does this mean that the branch must always be specified (i.e., no default branch from the context)?

Reply:
TBH, I did not understand this `MERGE BRANCH` at all. It seems awkward ("merge what?").